<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_Laser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>180</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Laser Select</string>
    </property>
    <widget class="QRadioButton" name="radioButton_L1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_L2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_L3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_L4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_SerialNum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Laser Serial Number</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_L4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_L3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_L2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_L1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_SerialNum_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>131</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fixture Number</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_F4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_F3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_F2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_F1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Distance">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>150</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Distance Select</string>
    </property>
    <widget class="QRadioButton" name="radioButton_D1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_D2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_LaserBeam">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LaserBeamAlignment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LaserPlane">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LaserPlaneAlignment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LaserVerification">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LaserVerification</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>721</width>
      <height>211</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Process steps:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1. Input serial num and fixutre num&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2. Select laser and distance D1&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3. Finish laserbeam alignment at D1, apply loctite, quit and save&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4. Move target to D2, open laserbeam alignment at D2, no adjustment is needed. Quit and save&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;5. Move target to D1. Put the Powell lens on and finish laserplane alignment at D1. Apply loctite. Quit and save&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;6. Move target to D2, open laserplane alignment at D2, no adjustment is needed. Quit and save&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;7. Click laserverification button before moving to next laser&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Double tab &amp;quot;q&amp;quot; to quit without saving&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Tab &amp;quot;q&amp;quot; then &amp;quot;s&amp;quot; to quit and save&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
